frontend/src/utils/util.ts

```diff
@@ -59,6 +59,26 @@ export function timezoneString(zoneName: string, offset: number): string {
   ).padStart(2, "0")}`;
 }
 
+export const UTC0ToLocal = (utc0Ts: number | null): number | null => {
```

Comment:
UTCToLocal, same in other places.

no need to mention 0 if we are just using UTC+0